GuiAutomation/AutomationRunner.cs

```diff
@@ -1185,7 +1185,7 @@ public static Task ShowWindowAndExecuteTests(SystemWindow initialSystemWindow, A
 				}));
 
 			// Once either the timeout or the test method has completed, store if a timeout occurred and shutdown the SystemWindow
-			task.ContinueWith((innerTask) =>
+			task.ContinueWith((Action<Task<Task>>)((Task<Task> innerTask) =>
```

Comment:
Should revert compiler generated weirdness back to simplified anonymous type definition

Reply:
Fixing